pkg/controllers/vdb/imageversion_reconciler.go

```diff
@@ -82,8 +83,7 @@ func (v *ImageVersionReconciler) Reconcile(ctx context.Context, _ *ctrl.Request)
 
 	vinf, ok := v.Vdb.MakeVersionInfo()
 	if !ok {
-		// Version info is not in the vdb.  Fine to skip.
-		return ctrl.Result{}, nil
+		return ctrl.Result{}, errors.New("version not found in the VerticaDB")
```

Comment:
Let's output the same  error message everywhere we check if the version is in the vdb.

Reply:
I'll create a variant of `MakeVersionInfo` that returns an error called `MakeVersionInfoCheck`